tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/LinkerBackendPlatformExtensions.scala

```diff
@@ -0,0 +1,30 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2017, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.core.tools.linker.backend
+
+import org.scalajs.core.tools.linker.backend.closure.ClosureLinkerBackend
+
+object LinkerBackendPlatformExtensions {
```

Comment:
No, there's no need to. The `implicit def` that converts a `LinkerBackend.Config` into a `LinkerBackendPlatformExtensions.ConfigExt` is defined in the companion object of `LinkerBackend.Config`. So even though the target type is not in the import scope, the implicit conversion itself is in the implicit scope, so the implicit pimp works without import. The same mechanism is already applied for `StandardConfig.closureCompiler`.